acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob

```diff
@@ -43,12 +58,12 @@ flake8:
 {{% endif %}}
 test:
   stage: test
-  tags:
-    - odoo-{{{ odoo.series }}}
   script:
     - virtualenv --python=$(which {{{ python_version }}}) venv
     - venv/bin/pip install coverage
-    - venv/bin/pip install -r requirements.txt -e .
+    - venv/bin/pip install --no-deps --no-index release/*.whl
+    # although the project is part of *.whl, install it in editable mode so coverage sees it
+    - venv/bin/pip install --no-deps --no-index -e .
```

Comment:
@ThomasBinsfeld could you try `venv/bin/pip install --no-index -e .`

My thinking: `--no-deps` does not check dependencies at all. Without `--no-deps` it will look for dependencies and will break when there are missing dependencies because of `--no-index`. So we detect dependency issues earlier.

Reply:
Indeed. I'll open a new PR.

Reply:
Done in https://github.com/acsone/acsoo/pull/31